scripts/zones/Mine_Shaft_2716/mobs/Bugboy.lua

```diff
@@ -0,0 +1,58 @@
+-----------------------------------
+-- Area: Mine Shaft 2716
+-- Mob: Bugboy
+-- ENM : Bionic Bug
+-----------------------------------
+mixins = { require('scripts/mixins/job_special') }
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.WEAPON_BONUS, 20)
+    xi.mix.jobSpecial.config(mob,
+    {
+        specials =
+        {
+            { id = xi.jsa.MIGHTY_STRIKES, cooldown = 70, hpp = 70 },
+        },
+    })
+end
+
+entity.onMobFight = function(mob, target)
+    if xi.combat.behavior.isEntityBusy(mob) then
+        return
+    end
+
+    -- 50% chance to follow up a TP move with a damaging TP move.
+    local skillUsed = mob:getLocalVar('skillUsed')
+    local followUpSkills =
+    {
+        xi.mobSkill.HEAVY_BLOW,
+        xi.mobSkill.HEAVY_WHISK,
+        xi.mobSkill.FLYING_HIP_PRESS,
+        xi.mobSkill.EARTH_SHOCK,
+    }
+    if
+        skillUsed ~= 0 and
```

Comment:
`skillUsed` is only used once and doesnt really help us visually, so theres no need for it, Just check the local entity variable directly